CONTRIBUTING.md

```diff
@@ -64,14 +64,16 @@ Go to the root of the repository.
 ```bash
 cd design-system
 yarn setup
+yarn develop
 ```
 
 #### 4. Start storybook for component and stories documentation changes
 
 Start the storybook application to test your changes to components or their documentation.
 
 ```bash
-yarn storybook
+cd docs
+yarn develop
```

Comment:
tbf if you run `yarn develop` it all runs (incl. storybook) we could omit step 4?

Reply:
oh yeah, cool. will do.